Webapp/sources/models/user.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, **kwargs) -> None:
         nullable=False,
     )
     time_of_creation = db.Column(db.DateTime)
+    last_login_time = db.Column(db.DateTime)
```

Comment:
This column wasn't in the PonyORM models, so I think this is intended as new functionality? If so - it will need a database migration. 
But we shouldn't add any further migrations until production is running on SQLAlchemy, as I want the initial migration just to reflect the existing data model to prevent any data loss.